### PR TITLE
Use TransferRequest of cuenca-validations and retrieve_uris

### PR DESCRIPTION
**Pull request by kerycdiaz**
Fix performance: Use TransferRequest of cuenca-validations and retrieve_uris

### CODE REVIEW

**Review by alexviquez — APPROVED** [review]
jaja el for, va !

File: cuenca/resources/resources.py [hunk 1]
```diff
@@ -18,6 +18,4 @@ def retrieve_uri(uri: str) -> Retrievable:
 
 def retrieve_uris(uris: List[str]) -> List[Retrievable]:
     with ThreadPoolExecutor(max_workers=len(uris)) as executor:
-        return [
-            obj for obj in executor.map(retrieve_uri, [uri for uri in uris])
-        ]
+        return [obj for obj in executor.map(retrieve_uri, uris)]
```
Comment (matin) [inline cuenca/resources/resources.py:21]: @kerycdiaz @alexviquez couldn't this be `list(executor.map(retrieve_uri, uris)`?

Comment (matin) [reply cuenca/resources/resources.py:21]:
```python
In [2]: list(range(10)) == [x for x in range(10)]
Out[2]: True
```

Comment (kerycdiaz) [reply cuenca/resources/resources.py:21]: Cierto Matin

**Pull request merged**